schemas/capability.json

```diff
@@ -461,14 +464,17 @@
       "not": {
         "anyOf": [
           { "required": ["effectIntensity", "effectIntensityStart"] },
-          { "required": ["speed", "speedStart"] }
+          { "required": ["speed", "speedStart"] },
+          { "required": ["duration", "durationStart"] }
```

Comment:
Isn't this one missing?

`{ "required": ["duration", "speed"] }` (plus all combinations of start / end)

Or will it be in the fixture test?

Reply:
It will be in the fixture test (see post above), like ShutterStrobe